genes/hgvs/hgvs_matcher.py

```diff
@@ -516,6 +516,8 @@ def clean_hgvs(self, hgvs_string) -> Tuple[str, List[str]]:
         cleaned_hgvs = clean_string(hgvs_string)  # remove non-printable characters
         cleaned_hgvs = cleaned_hgvs.replace(" ", "")  # No whitespace in HGVS
         cleaned_hgvs = cleaned_hgvs.replace("::", ":")  # Fix double colon
+        if cleaned_hgvs.startswith("M_") or cleaned_hgvs.startswith("m_"):
```

Comment:
Really minor coding style preference:
Could you change this to
`if cleaned_hgvs[0:2].upper() == "M_"`
to avoid making a check for M_ and m_. Just saves us repeating ourselves a little bit.